### DISCUSSION TITLE
Regarding Recurring Update Reminders, Label and Meta File Synchronization Emails

### DISCUSSION THREAD

**Discussion by firatbezir** [WordPress/openverse#3686] [Q&A] [answered]
Hello everyone,

I’ve been contributing to this project and have made several PRs that have been merged and issues that have been closed. However, I’ve been receiving three emails daily with the following subjects:

1. Project Thread Update Reminders #47
2. Label sync #47
3. Meta file sync #51

I’m not sure why I’m receiving these emails and what they signify. I’ve attached screenshots of these emails below for reference.

![image](https://github.com/WordPress/openverse/assets/51786059/b4248048-d34c-482f-b348-96b1d2367c17)
![image](https://github.com/WordPress/openverse/assets/51786059/6bff1875-9aa2-467a-9c49-24e6f7aaa88c)
![image](https://github.com/WordPress/openverse/assets/51786059/e95c257a-06a8-4862-bd1e-a7e3f7236a45)

Could anyone please explain why I might be receiving these emails and what actions, if any, I need to take? Any help would be greatly appreciated.

Thank you in advance for your time and assistance.

**Comment by dhruvkb** [comment, marked as answer]
These workflows are not relevant for forks and they should not run for forked repos. We'll make an issue for stopping the workflows on forks and prevent them from running. In the meantime, @firatbezir you can control what emails you receive in GitHub Settings, under [Notification settings](https://github.com/settings/notifications).

**Reply by firatbezir** [reply]
Thank you @dhruvkb

**Reply by dhruvkb** [reply]
The issue has been filed @ #3693.

**Comment by krysal** [comment]
@firatbezir Thanks a lot for sharing this with us! I had no idea this was happening, these problems should definitely be avoided.
